src/elm/clinical.coffee

```diff
@@ -27,6 +27,24 @@ module.exports.ValueSetRef = class ValueSetRef extends Expression
       valueset = valueset.execute(ctx)
     valueset
 
+module.exports.AnyInValueSet = class AnyInValueSet extends Expression
+  constructor: (json) ->
+    super
+    @codes = build json.codes
+    @valueset = new ValueSetRef json.valueset
+
+  exec: (ctx) ->
+    valueset = @valueset.execute(ctx)
+    # If the value set reference cannot be resolved, a run-time error is thrown.
+    throw new Error("ValueSet must be provided to InValueSet function") unless valueset? and valueset.isValueSet
+
+    codes = @codes.exec(ctx)
+    for code in codes
+      # If the code argument is null, the result is false.
+      return false unless code?
```

Comment:
I see the spec says that if the code argument is null, the result is false. But I think at that point the spec is referring to when the argument is one code, not a list. Are you sure it makes sense to return false if any code in the list is null? I think it might be better to ignore the null codes.